spinner/eglspinner.cpp

```diff
@@ -317,7 +335,41 @@ try
     }
     g_clear_error(&error);
 
-    auto const surfaces = mir_eglapp_init(argc, argv);
+    std::vector<std::shared_ptr<EglSurface>> surfaces;
+    char* backendEnv = getenv("COMPOSITOR_BACKEND");
+    Backend backend = UNKOWN;
+
+    if (backendEnv == nullptr) {
+        if (session_config.is_android()) {
+            printf("Is android device, defaults to mir\n");
+            backend = MIR;
+        } else {
+            printf("Is not android device, defaults to wayland\n");
```

Comment:
Consider the following:

"Running on an Android device, using Mir output."

"Not running on an Android device, using Wayland output."